lib/stdlib/src/erl_stdlib_errors.erl

```diff
@@ -1069,6 +1075,8 @@ expand_error(not_atom) ->
     <<"not an atom">>;
 expand_error(not_binary) ->
     <<"not a binary">>;
+expand_error(bad_hex_case) ->
+    <<"not uppercase or lowercase">>;
 expand_error(not_compiled_regexp) ->
```

Comment:
> @gilbertwong96
> 
> > However, when I took a quick look yesterday and ran the test suite and attempted to build the documentation, I noticed that the test case `binary_module_SUITE:error_info/1` fails, and that the documentation fails to build.
> 
> The documentation build failure is probably because you removed the `since` tag from your new function altogether. I believe it can be left empty, but its presence is required. Since you now have a ticket number, you should put it in the `since` tag like this `since="OTP @OTP-18354@"`, and that should also solve the build issue.
> 
> About the `error_info` test failure, I have no idea though =^^=

I have fixed the `error_info` test failure now.  @juhlig